core/templates/pages/exploration-player-page/services/hints-and-solution-manager.service.ts

```diff
@@ -42,7 +46,10 @@ export class HintsAndSolutionManagerService {
   solutionReleased: boolean = false;
   solutionConsumed: boolean = false;
   hintsForLatestCard: Hint[] = [];
-  solutionForLatestCard: Solution = null;
+  // This in initialized using the the class methods
+  // and we need to do non-null assertion, for more information see
+  // https://github.com/oppia/oppia/wiki/Guide-on-defining-types#ts-7-1
+  solutionForLatestCard!: Solution;
```

Comment:
Should this be moved up? It seems to me that usually, we place this as up as possible.